code/obj/item/implant.dm

```diff
@@ -1099,20 +1152,23 @@ THROWING DARTS
 
 /obj/item/implanter/mindslave
 	icon_state = "implanter1-g"
+	only_in = "chest"
```

Comment:
Still a noob trap. Why is the variable set at all for all this stuff?

Reply:
For the mindslaver implant, I guess it would be fine to put them in any limb. `only_in` is mainly for microbombs, to prevent people from cutting their own legs off to detonate the mindslave implant. I suppose it could go, now that microbombs just never go off inside of limbs.